src/ocean/LibFeatures.d

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+
+    Meta module defining constants that match ocean feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later ocean versions.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.LibFeatures;
+
+const has_features_2_5 = true;
+const has_features_2_4 = true;
```

Comment:
> Yes, this is what I mean, for me this is less misleading, because in v2.4 there will be no feature constant so anyone using v2.4 will not get features from it used (simply because the feature constant will not be defined)

OK, here lies the point that I was missing. In v2.4 the new function is not defined so it simply won't compile. I still think is not ideal but I guess for the transitional period there is no ideal solution, so we can go as well with this one.